xmppserver/src/main/java/org/jivesoftware/openfire/server/ServerDialback.java

```diff
@@ -266,6 +267,8 @@ public LocalOutgoingServerSession createOutgoingSession(String localDomain, Stri
                     // Set the hostname as the address of the session
                     session.setAddress(new JID(null, remoteDomain, null));
                     log.debug( "Successfully created new outgoing session!" );
+                    // After the session has been created, inform all listeners as well.
+                    ServerSessionEventDispatcher.dispatchEvent(session, ServerSessionEventDispatcher.EventType.session_created);
```

Comment:
This looks OK - but don't forget to also dispatch an event when the session is destroyed!